movieland-persistence-jdbc/src/main/java/com/gromoks/movieland/dao/JdbcReviewDao.java

```diff
@@ -1,26 +1,41 @@
 package com.gromoks.movieland.dao;
 
+import com.gromoks.movieland.dao.mapper.ReviewRowMapper;
+import com.gromoks.movieland.entity.Movie;
 import com.gromoks.movieland.entity.Review;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
 import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
 import org.springframework.jdbc.support.GeneratedKeyHolder;
 import org.springframework.jdbc.support.KeyHolder;
 import org.springframework.stereotype.Repository;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @Repository
 public class JdbcReviewDao implements ReviewDao {
```

Comment:
ReviewDao --> IReviewDao would be better

Reply:
Use other convension for implementation like JdbcReviewDaoImpl. It was missed here but consider it in separate hot fix branch